webpack.common.config.ts

```diff
@@ -200,6 +203,24 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false
     }),
+    new HtmlWebpackPlugin({
+      chunks: ['diagnostic_test_player_page'],
+      filename: 'diagnostic-test-player-page.mainpage.html',
+      hybrid: true,
+      meta: {
+        name: defaultMeta.name,
+        description: 'With Oppia, you can access free lessons on ' +
```

Comment:
This probably needs a more page-specific description -- it's a potential landing page, right?

Reply:
Added, PTAL